### ISSUE TITLE
Sync Image and color transition of header.

### ISSUE THREAD

**Issue by helium9** [Web-Team-IITI-Gymkhana/gymkhana-website#1]
Can we sync the image and color transition together of the header section?

**Comment by mittal-ishaan** [comment]
yes we can. the image animation is controlled by slider keyframe and color transition is controlled by animate keyframe in style.css. though we have kept their transition time same, as the image is loaded afterwards, the sync is disturebed. we will look into this at a later stage.
